src/btree/bt_debug.c

```diff
@@ -641,12 +642,17 @@ __debug_cell_kv(
     /* Row-store references to empty cells return a NULL on-page reference. */
     if (unpack->cell == NULL)
         return (__debug_item(ds, tag, "zero-length", strlen("zero-length")));
-
-    if (F_ISSET(ds, WT_DEBUG_UNREDACT))
+    if (F_ISSET(ds, WT_DEBUG_UNREDACT_ALL))
         WT_RET(ds->f(ds, "\t%s: len %" PRIu32, __wt_cell_type_string(unpack->raw), unpack->size));
-    else
-        WT_RET(ds->f(ds, "\t%s: {REDACTED}", __wt_cell_type_string(unpack->raw)));
-
+    else if (F_ISSET(ds, WT_DEBUG_UNREDACT_KEYS)) {
+        if (unpack->raw == WT_CELL_KEY || unpack->raw == WT_CELL_KEY_PFX ||
+          unpack->raw == WT_CELL_KEY_OVFL || unpack->raw == WT_CELL_KEY_SHORT ||
+          unpack->raw == WT_CELL_KEY_SHORT_PFX || unpack->raw == WT_CELL_KEY_OVFL_RM)
+            WT_RET(
+              ds->f(ds, "\t%s: len %" PRIu32, __wt_cell_type_string(unpack->raw), unpack->size));
+        else
```

Comment:
Should this else be on line 656?

Reply:
yes i believe it should be in both spots, fixed https://github.com/wiredtiger/wiredtiger/pull/10094/commits/58db06afbde781bb330bbe8aabb606d2a2071972